### PR TITLE
SURF-549 Doc edits for app nav

### PR DESCRIPTION
**Pull request by robb-romans**
Closes SURF-549

### CODE REVIEW

File: _docs/layout/navigation.md [hunk 1]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
 
-Navigation establishes where the user is in an application and provides a path
-to complete tasks. Consistent navigation patterns allow the user to complete
-tasks more efficiently and confidently. Rackspace applications may utilize a
-combination of eyebrow navigation, either a top or side navigation, and
-breadcrumbs.
+Navigation establishes a user's location within an application and
+shows the path forward to accomplish a goal. Consistent navigation
+patterns increase user confidence and efficiency. You can use a
+combination of eyebrow navigation, either top or side navigation, and
```
Comment (asettle) [inline _docs/layout/navigation.md]: So, I wonder if we should remove 'eyebrow navigation'? There will be users outside of designers that will be accessing this content. It's not necessarily colloquial, but it's not exactly common either. @palindromebart - can we think about this?

File: _docs/layout/navigation.md [hunk 2]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
+breadcrumbs to guide users.
```
Comment (asettle) [inline _docs/layout/navigation.md]: ditto with breadcrumbs.

File: _docs/layout/navigation.md [hunk 3]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
 
 {% include toc.html %}
 
-# Primary Navigation {% include complete.html %}
+# Primary navigation {% include complete.html %}
 
-The primary navigation is used for the main sections of the application or for
-dropdown categories that organize sections into higher level groups.
+Use primary navigation for the main sections of an application or for
```
Comment (asettle) [inline _docs/layout/navigation.md]: application, or for * (but that could probably be debated)

File: _docs/layout/navigation.md [hunk 4]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
 
 <div class="row">
 {% column left:"col-md-4" %}
-The Left Navigation component is used on all web applications that desire to
-implement a highly compliant version of the Helix Design Language.
 
--   Can extend to fit any navigation IA via scrolling when the list of
-    navigational items exceeds the screen view-height.
+Use the left navigation component for web applications when you want
+to implement a highly-compliant version of the Helix design language.
```
Comment (asettle) [inline _docs/layout/navigation.md]: What do we mean by 'highly-compliant' ? Without context (other than 'version'), this could mean many things.

File: _docs/layout/navigation.md [hunk 5]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
 
--   Preferred implementation includes only primary and secondary navigational
-    links.
+-   This component can extend to fit any navigation information
+    architecture (IA) by scrolling when the list of navigation items
+    exceeds the screen view height. In other words, if there are many
```
Comment (asettle) [inline _docs/layout/navigation.md]: I'd remove "in other words" - I wonder if it's just worth keeping the second sentence, since it is in layman, rather than the first full explanation?

File: _docs/layout/navigation.md [hunk 6]
```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
...
+    options to display, the left navigation is scroll-able so that
+    users have access to all items.
 
-{% include note.html content="A less preferred implementation that includes [tertiary navigational](#tertiary-left-navigation) elements is also provided."%}
+-   The preferred implementation includes only primary and secondary
+    navigation links.
+
+-   An alternative, non-preferred implementation includes
```
Comment (asettle) [inline _docs/layout/navigation.md]: non-preferred implemented, includes*

File: _docs/layout/navigation.md [hunk 7]
```diff
@@ -135,65 +149,70 @@ for highly compliant Helix applications is the [left navigation](#side-nav).
...
 
--   Each menu item button has 20px left and right padding
+-   Horizontal spacing includes 20px of left margin.
+
+-   Each menu item button has 20px of left and right padding.
 
-    {% include note.html content="Dropdown padding left begins at the left
-    border of the parent primary navigational item and padding right begins
-    after the longest textual element in the dropdown element." %}
+    {% include note.html content="Drop-down element left padding
```
Comment (asettle) [inline _docs/layout/navigation.md]: "Drop-down element left padding" - I'm not at all sure I understand this. Will others?

File: _docs/layout/navigation.md [hunk 8]
```diff
@@ -233,116 +253,124 @@ navigational element.
 
 # Eyebrow {% include inprogress.html %}
 
-The eyebrow sits as the top most menu bar across all breakpoints within the
-Application Frame of a Helix interface. It provides access to platform level
-services as well as aggregate information across a users accounts at Rackspace.
-The eyebrow pattern is often referred to internally by the project name "Pilot
-Nav" or by the function it serves "Utility Nav".
+The eyebrow is the top menu bar across all breakpoints within the
```
Comment (asettle) [inline _docs/layout/navigation.md]: eyebrow - see comment above

File: _docs/layout/navigation.md [hunk 9]
```diff
@@ -233,116 +253,124 @@ navigational element.
...
+application frame of a Helix interface. The eyebrow provides access to
+platform-level services and aggregate information across a user's
+accounts at Rackspace. The eyebrow pattern is often referred to
+internally as "pilot nav" or "utility nav".
```
Comment (asettle) [inline _docs/layout/navigation.md]: Since it's MD, do we want to do *pilot nav* (with the * WORD * )

File: _docs/layout/navigation.md [hunk 10]
```diff
@@ -233,116 +253,124 @@ navigational element.
...
+content area and navigation section changes based on the selected
+state beneath it.
 
 <div class="row">
 {% column left:"col-md-4" %}
--   The Rackspace® logo serves as a button to trigger the global dashboard.
 
--   Product name to the left of the logo serves as wayfinding to show the
-    currently selected product.
+-   The Rackspace® logo  button triggers the global dashboard.
```
Comment (asettle) [inline _docs/layout/navigation.md]: Is "Rackspace®" the logo? Or is there a Rackspace logo that is referenced somewhere else?

File: _docs/layout/navigation.md [hunk 11]
```diff
@@ -233,116 +253,124 @@ navigational element.
...
 
--   Leftside dropdowns provide links to product lines and NEVER include a
-    secondary CTA
+-   A user can switch product lines by clicking the product button,
+    then clicking a drop-down element.
 
--   Leftside dropdowns do not have a defined disabled state. Only menu items
-    to which the currently logged in user has access will appear
+-   Drop-down elements on the left side provide links to product lines
+    and never include a secondary CTA.
```
Comment (asettle) [inline _docs/layout/navigation.md]: CTA? Am I missing something?

File: _docs/layout/navigation.md [hunk 12]
```diff
@@ -233,116 +253,124 @@ navigational element.
...
-### Eyebrow (right side) Dropdowns
+### Eyebrow (right side) drop-down elements
 
 <div class="row">
 {% column left:"col-md-4" %}
 
--   Items within the right-side of the eyebrow produce either a dropdown or
-    display an off-canvas panel onClick
+-   Items within the right side of the eyebrow show either a drop-down
+    menu or an off-canvas panel onClick.
```
Comment (asettle) [inline _docs/layout/navigation.md]: onClick i'm guessing is the name of the panel? Should this then be marked up? *onClick* ?

File: _docs/layout/navigation.md (hunk 3 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: Adding the comma is fine. This is a "f.a.n.b.o.y.s." situation isn't it?

File: _docs/layout/navigation.md (hunk 1 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: Rather than remove it, let's document the term in our upcoming glossary/terms and conditions page

Comment (asettle) [inline _docs/layout/navigation.md]: Works for me

File: _docs/layout/navigation.md (hunk 4 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: This is another term that should be documented elsewhere: either a page about compliance or glossary.

File: _docs/layout/navigation.md (hunk 6 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: don't think extra comma needed here

File: _docs/layout/navigation.md (hunk 7 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: Perhaps we need to cover padding in a separate topic, or glossary?

File: _docs/layout/navigation.md (hunk 3 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: Adding.

File: _docs/layout/navigation.md (hunk 7 above)
Comment (asettle) [inline _docs/layout/navigation.md]: Might be a good idea

File: _docs/layout/navigation.md (hunk 9 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: agreed, will change

File: _docs/layout/navigation.md (hunk 10 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: not sure

File: _docs/layout/navigation.md (hunk 11 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: CTA needs to be defined in glossary.

File: _docs/layout/navigation.md (hunk 12 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: don't know. @palindromebart any guidance?

File: _docs/layout/navigation.md (hunk 10 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: @palindromebart ? ^^^

Comment (asettle) [inline _docs/layout/navigation.md]: @palindromebart - could you clarify this one, please? :)

File: _docs/layout/navigation.md (hunk 1 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: @asettle what about we define eyebrow? e.g. You can combine global navigation (via the eyebrow), application navigation (via the side/top navigation), and breadcrumbs.

File: _docs/layout/navigation.md (hunk 2 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: see above

File: _docs/layout/navigation.md (hunk 4 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: Since we are moving all applications to using a side navigation, maybe change the sentence to:
Use the left navigation component for you web application unless there is a firm implementation constraint that require you to use a top navigation.

File: _docs/layout/navigation.md (hunk 5 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: I agree, keep the simpler version.

File: _docs/layout/navigation.md (hunk 7 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: this could use laymans terms
e.g. 
The left padding of a dropdown begins at the left border of the parent primary navigational item and padding right begins after the longest textual element in the dropdown element.

File: _docs/layout/navigation.md (hunk 11 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: CTA= call to action

File: _docs/layout/navigation.md (hunk 12 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: no. I believe onClick is how one defines the trigger type that initiates the defined behavior. we can separate that into regular language to on mouse click if @technabors doesn't object.

File: _docs/layout/navigation.md (hunk 10 above)
Comment (palindromebart) [inline _docs/layout/navigation.md]: The eyebrow has a [fanatiguy icon] + the word [rackspace] in the left corner. this is treated as a single "logo icon". when a user clicks on it, they will trigger the global dashboard to open. 
The product selector is immediately next to the "logo icon" (on the right side)

File: _docs/layout/navigation.md (hunk 1 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: I'll use a that verbiage, slightly tweaked to remove the Latin.

File: _docs/layout/navigation.md (hunk 4 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: k

File: _docs/layout/navigation.md (hunk 7 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: "dropdown" is not a noun; however, "drop-down" describes a "thing" such as element or menu. Also, I'd prefer using the simpler `navigation` item versus `navigational` item.
Revised to read:
```
The left padding of a drop-down element begins at the left border 
of the parent primary navigation item and the right padding begins 
after the longest text element in the drop-down element.
```

File: _docs/layout/navigation.md (hunk 12 above)
Comment (robb-romans) [inline _docs/layout/navigation.md]: Changed to `Items within the right side of the eyebrow show either a drop-down menu or an off-canvas panel when clicked.`

**Comment by robb-romans** [comment]
@asettle @palindromebart updated. please review/approve/merge. thanks.

**Comment by robb-romans** [comment]
@palindromebart @technabors please review and merge. thanks.
